queues.md

```diff
@@ -151,6 +151,7 @@ The following dependencies are needed for the listed queue drivers. These depend
 - Amazon SQS: `aws/aws-sdk-php ~3.0`
 - Beanstalkd: `pda/pheanstalk ~5.0`
 - Redis: `predis/predis ~2.0` or phpredis PHP extension
+- MongoDB: `mongodb/laravel-mongodb`, read more about [Laravel Queue with MongoDB](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)
```

Comment:
Per my earlier comment about MongoDB not belonging in the existing paragraph's list, this may be the only addition needed on this page. In any event, I have no criticism of the addition here.

Reply:
Feel free to resolve.